### DISCUSSION TITLE
1979 iverson paper

### DISCUSSION THREAD

**Discussion by justin2004** [phantomics/april#174] [General]
there are some errata and incorrect OCR'd text in the PDF (linked on April's readme):
https://www.eecg.utoronto.ca/~jzhu/csc326/readings/iverson.pdf

but this source seems pretty good (and you can copy the expressions into a clipboard):
https://www.jsoftware.com/papers/tot.htm

thanks to this user:
https://www.reddit.com/r/apljk/comments/mupbox/iverson_1979_paper_vector_permutation_not_working/gv91drm/

just noting this in case you want the readme to reference that source.

**Comment by phantomics** [comment]
Thanks, readme will be updated at next commit.

**Reply by phantomics** [reply]
This has been updated.
